Dangerfile

```diff
@@ -45,11 +45,8 @@ def check_pr_size_increase
   bypass_size_check = github.pr_labels.any? { |s| s == BYPASS_LABEL }
 
   if bypass_size_check
-    warn("Size check is being bypassed due to the presence of the label \"#{BYPASS_LABEL}\"")
```

Comment:
I think this warning is still valid? It indicates the check is being bypassed

Reply:
Yeah, I was thinking about it but I thought that if you've added the label, you don't need to be notified? I guess we could still add it to the warning if the size is more than the warn threshold... Will move it there instead. Lmk if you think otherwise!

Reply:
It's fine for me either way, I was just wondering if you were removing it on purpose, since it seemed not strictly the main change of the PR. You can leave the change 🙂